src/encoded/audit/experiment.py

```diff
@@ -2902,44 +2902,65 @@ def audit_experiment_no_processed_data(value, system, files_structure):
         yield AuditFailure('lacking processed data', detail, level='WARNING')
 
 
-def audit_experiment_inconsistent_analysis_files(value, system, files_structure):
+def audit_experiment_inconsistent_analysis_files_mismatched_dataset(value, system, files_structure):
+    files_in_different_dataset = []
+    if 'analyses' not in value:
+        return
+
+    for analysis in value['analyses']:
+        if analysis['status'] == 'deleted':
+            continue
+        for file in analysis['files']:
+            if file['dataset'] != value['@id']:
+                files_in_different_dataset .append(file['@id'])
```

Comment:
it looks like there is a space between "files_in_different_dataset" and ".append"